src/chrome/content/rules/Irish-Country-Cottages.co.uk.xml

```diff
@@ -0,0 +1,16 @@
+<!--
+	For other Wyndham related rulesets, see Wyndham.xml
+
+	Non-functional hosts
+		Timeout was reached:
+		- irish-country-cottages.co.uk
+
+		Mixed content blocking (MCB) triggered:
+		- www.irish-country-cottages.co.uk
+-->
+<ruleset name="Irish-Country-Cottages.co.uk" platform="mixedcontent" default_off="timeout">
```

Comment:
`platform="mixedcontent"` and `default_off="timeout"` at the same time makes no sense. PLease remove the `default_off`and redirect `^` to `www`.

Reply:
fixed. thanks!!